packages/react-components/react-dialog/src/stories/DialogDefault.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { Dialog, DialogTrigger, DialogSurface, DialogTitle, DialogBody, DialogActions } from '@fluentui/react-dialog';
+import type { DialogProps } from '@fluentui/react-dialog';
+import { Button } from '@fluentui/react-components';
+
+export const Default = (props: Partial<DialogProps>) => {
```

Comment:
```suggestion
export const Default = () => {
```

We no longer use storybook controls in our docsite, and this code leaks into our codesandbox exports, so I don't think should include this in stories anymore